packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.tsx

```diff
@@ -1,35 +1,34 @@
-import React, { CSSProperties, useCallback, useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import classNames from 'classnames';
 import { useCombobox } from 'downshift';
 import ArrowIcon from '../../../public/images/pointed-down-arrow-icon.svg';
 import reactNodeToString from '../../../utils/reactNodeToString';
 import { TGenericSizes } from '../types';
 import { WalletText } from '../WalletText';
+import { WalletTextField } from '../WalletTextField';
+import { WalletTextFieldProps } from '../WalletTextField/WalletTextField';
 import './WalletDropdown.scss';
 
-type TProps = {
+type TProps = Pick<WalletTextFieldProps, 'id' | 'label' | 'value'> & {
```

Comment:
use name instead of id and passed it accordingly

Reply:
Updated